### PR TITLE
fix: [SRM-12500]: ELk bugs hotfix

### PR DESCRIPTION
**Pull request by vasuki-kalluri**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "nextgenui",
   "description": "Harness Inc",
-  "version": "0.324.0",
+  "version": "0.324.3",
```
Comment (praneshg239) [inline package.json:4]: why `0.324.0` to `0.324.3`?

Also, already we have `0.324.3` in the branch. 
https://github.com/harness/harness-core-ui/blob/release/0.324.x/package.json#L4

Please do rebase and update to `0.342.4`

**Review by praneshg239 — CHANGES_REQUESTED** [review]
please check

File: package.json (hunk 1 above)
Comment (vasuki-kalluri) [reply package.json]: Done

**Comment by vasuki-kalluri** [comment]
retrigger coverage

**Review by praneshg239 — APPROVED** [review]

**Pull request merged**